tests/integrations/celery/test_celery.py

```diff
@@ -47,9 +48,15 @@ def dummy_task(x, y):
         foo = 42  # noqa
         return x / y
 
+    span_context = SpanContext.start_trace()
```

Comment:
Is there a reason why we just can do "new SpanContext"?
The ctor can set the values, since we are not measuring time, this can simplify the API.

Reply:
For prosperity from a conversation in person: Mostly just so that the ctor will not attempt to auto generate. We can change this, that was just an early draft.